libtinyfiledialogs/tinyfiledialogs.c

```diff
@@ -209,6 +209,14 @@ char const tinyfd_needs[] = "\
 #pragma warning(disable:4706) /* allows usage of strncpy, strcpy, strcat, sprintf, fopen */
 #endif
 
+#ifdef _WIN64
+#include <ShellScalingApi.h>
+
+#pragma comment(linker,"\"/manifestdependency:type='win32' \
```

Comment:
Have you verified this against a non-MSVC toolchain?

Reply:
No